.travis.yml

```
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+before_install:
+  - cd web-components
+
+script:
+  - npm run build
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $repo_token
+  local_dir: web-components/build
```

Тоже для второго дз сборка
```suggestion
  local_dir: build
```